lib/streams.ts

```diff
@@ -47,32 +49,26 @@ class ReadStream {
 			options = {buffer: optionsOrStreamLike};
 		} else if (optionsOrStreamLike instanceof Buffer) {
 			options = {buffer: optionsOrStreamLike};
-		} else if (typeof /** @type {any} */ (optionsOrStreamLike)._readableState === 'object') {
-			options = {nodeStream: /** @type {NodeJS.ReadableStream} */ (optionsOrStreamLike)};
+		} else if (typeof (optionsOrStreamLike as any)._readableState === 'object') {
+			options = {nodeStream: optionsOrStreamLike as NodeJS.ReadableStream};
 		} else {
 			options = optionsOrStreamLike;
 		}
 		if (options.nodeStream) {
-			const nodeStream = /** @type {NodeJS.ReadableStream} */ (options.nodeStream);
+			const nodeStream: NodeJS.ReadableStream = options.nodeStream;
 			this.nodeReadableStream = nodeStream;
 			nodeStream.on('data', data => {
 				this.push(data);
 			});
 			nodeStream.on('end', () => {
 				this.push(null);
 			});
-			/**
-			 * @this {ReadStream}
-			 * @param {number} bytes
-			 */
-			options.read = function (bytes) {
+
+			options.read = function (unusedBytes: number) {
```

Comment:
```suggestion
			options.read = function (this: ReadStream, unusedBytes: number) {
```

Reply:
VSCode was complaining why I did this, but I misinterepreted it being about `this` and not about `null`ness. Fixed.

Reply:
Wait, what does that mean?